elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/metadata/SQLReferenceTable.java

```diff
@@ -194,6 +244,56 @@ public void resolveAndStoreAllReferencesAndJoins(Queryable queryable) {
         return projections;
     }
 
+    private void populateTableContext(Type<?> entityClass, TableContext tableCtx, String prefixPath,
+                    Set<Type<?>> visitedEntities) {
+
+        Set<Type<?>> entities = new HashSet<Type<?>>(visitedEntities);
+        entities.add(entityClass);
+
+        SQLTable table = (SQLTable) metaDataStore.getTable(entityClass);
+        table.getColumns().forEach(column -> {
+            tableCtx.put(column.getName(), new ColumnContext(column.getExpression(),
+                                                             prefixPath,
+                                                             prepareColumnArgsMap(column.getArguments(), COL_PREFIX)));
```

Comment:
I don't understand the $$column prefix here.  Shouldn't a ColumnContext not care about its handlebars prefix?

Reply:
Removed it.